subworkflows/local/utils_nfcore_rnavar_pipeline/main.nf

```diff
@@ -81,23 +81,23 @@ workflow PIPELINE_INITIALISATION {
         params.star_index
     ]
 
-    // only check if we are using the annotate_tools
-    if (params.annotate_tools && (params.annotate_tools.split(',').contains('snpeff') || params.annotate_tools.split(',').contains('merge'))) checkPathParamList.add(params.snpeff_cache)
-    if (params.annotate_tools && (params.annotate_tools.split(',').contains('vep')    || params.annotate_tools.split(',').contains('merge'))) checkPathParamList.add(params.vep_cache)
+    // only check if we are using the tools
+    if (params.tools && (params.tools.split(',').contains('snpeff') || params.tools.split(',').contains('merge'))) checkPathParamList.add(params.snpeff_cache)
```

Comment:
I don't think this is used anymore since nf-schema does path validation

Reply:
ok, let's a cleanup in a following PR before the release